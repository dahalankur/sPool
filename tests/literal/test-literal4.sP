# string
"Hello World!"
"a nice friendly string"
"multiple \n lines \n done \n properly \n\n\n"
"multiple quotes escaped well!" # todo fix???
"dfjaskld;fjas;djfwieorpuewtpo324095743806943807riewofsn,nxzmxn~~```12345678)(*&^%$#@!!@#$%^&*()-={]\}{}|;<>?/.,/'"
