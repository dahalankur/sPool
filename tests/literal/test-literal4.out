PROGRAM {[ 
    STRING("Hello World!") 
    STRING("a nice friendly string") 
    STRING("multiple \n lines \n done \n properly \n\n\n") 
    STRING("multiple \"\"\" quotes \"\"\" escaped well!") 
    STRING("dfjaskld;fjas;djfwieorpuewtpo324095743806943807riewofsn,nxzmxn~~```12345678)(*&^%$#@!!@#$%^&*()-={]\}{}|;<>?/.,/'")]}
